JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java

```diff
@@ -144,8 +145,21 @@ public void processRequestParameters(DerivationInputs derivationInputs, Derivati
         }
 
         if (citiesNamespace != null) {
-            // AERMOD will be run for flat terrain in the dev-aermod-agent-cleanup branch.
-            // queryClient.setElevation(staticPointSources, buildings, srid);
+            if (queryClient.tableExists()) {
+                queryClient.setElevation(staticPointSources, buildings, srid);
+                List<byte[]> elevData = queryClient.getScopeElevation(scope, srid);
```

Comment:
alternatively you could have a try-with-resource block here and pass the connection object into both methods